src/tools/annotation/RectangleRoiTool.js

```diff
@@ -35,8 +35,8 @@ export default class RectangleRoiTool extends BaseAnnotationTool {
       name: 'RectangleRoi',
       supportedInteractionTypes: ['Mouse', 'Touch'],
       configuration: {
-        // showMinMax: false,
-        // showHounsfieldUnits: true
+        // ShowMinMax: false,
+        // ShowHounsfieldUnits: true
```

Comment:
Ideally, we would preserve these to show what configuration options exist. Forcing the first letter capital in these comments provides an incorrect key/prop.